src/main/java/BasicMAPF/Solvers/PrioritisedPlanning/PrioritisedPlanning_Solver.java

```diff
@@ -336,12 +334,23 @@ private static InstanceReport initSubproblemReport(MAPF_Instance instance) {
 
     protected RunParameters getSubproblemParameters(MAPF_Instance subproblem, InstanceReport subproblemReport, ConstraintSet constraints, float maxCost, Solution solutionSoFar) {
         long timeLeftToTimeout = Math.max(super.maximumRuntime - (System.nanoTime()/1000000 - super.startTime), 0);
-        RunParameters_SAAStar params = new RunParameters_SAAStar(new RunParametersBuilder().setTimeout(timeLeftToTimeout).
-                setConstraints(new ImmutableConstraintSet(constraints)).setInstanceReport(subproblemReport).setAStarGAndH(this.singleAgentGAndH).createRP());
+
+        RunParameters_SAAStar params;
+        if (transientMAPFBehaviour == TransientMAPFBehaviour.transientMAPFsst) {
+            params = new RunParameters_SAAStar(new RunParametersBuilder().setTimeout(timeLeftToTimeout).
+                    setConstraints(new ImmutableConstraintSet(constraints)).setInstanceReport(subproblemReport).setAStarGAndH(new ServiceTimeGAndH(this.agents, subproblem.map)).createRP());
+        }
+        else {
+            params = new RunParameters_SAAStar(new RunParametersBuilder().setTimeout(timeLeftToTimeout).
+                    setConstraints(new ImmutableConstraintSet(constraints)).setInstanceReport(subproblemReport).setAStarGAndH(this.singleAgentGAndH).createRP());
+        }
+
+//        RunParameters_SAAStar params = new RunParameters_SAAStar(new RunParametersBuilder().setTimeout(timeLeftToTimeout).
+//                setConstraints(new ImmutableConstraintSet(constraints)).setInstanceReport(subproblemReport).setAStarGAndH(this.singleAgentGAndH).createRP());
         params.fBudget = maxCost;
         if (transientMAPFBehaviour == TransientMAPFBehaviour.transientMAPF){
             params.goalCondition = new VisitedTargetAStarGoalCondition();
-        } else if (transientMAPFBehaviour == TransientMAPFBehaviour.transientMAPFWithBlacklist) {
+        } else if (transientMAPFBehaviour == TransientMAPFBehaviour.transientMAPFWithBlacklist || transientMAPFBehaviour == TransientMAPFBehaviour.transientMAPFsst) {
```

Comment:
agreed, but rename `transientMAPFsst` to `transientMAPFsstWithBlacklist`.
It's getting complicated... May have to make a "transient settings" sort of class if we end up adding more to this